include/mav/Message.h

```diff
@@ -440,7 +478,17 @@ namespace mav {
             return ss.str();
         }
 
-        [[nodiscard]] uint32_t finalize(uint8_t seq, const Identifier &sender) {
+        void sign(const std::array<uint8_t, 32>& key, const uint64_t& timestamp) {
+            signature().linkId() = 0;
```

Comment:
`linkId == 0` is the only option we ever care about?

Reply:
In our implementation, yes. We're not using the link ids, and it also breaks down a bit given some of the changes in RAS-A that identify a vehicle via a network address instead of system Id. I'm not even sure how that identification would work for a vehicle having multiple links. 

Let me know if you'd prefer otherwise, but I'm just going to add an additional argument to pass in the link id that defaults to 0.